recipes/NAS/orion_search.py

```diff
@@ -0,0 +1,564 @@
+import logging
+import os
+import time
+from collections import OrderedDict
+import matplotlib.pyplot as plt
+import timm
+import torch
+import torch.nn as nn
+import torchinfo
+import sys
+import numpy as np
+sys.path.append("../")
```

Comment:
this is not ideal, as it hides some possible bugs. I guess we need this to call stuff from the image_classification recipe?